requirements.txt

```diff
@@ -7,30 +7,31 @@
 -e file:.#egg=flytekit    # via -r requirements.in
 ansiwrap==0.8.4           # via papermill
 appdirs==1.4.4            # via black
-appnope==0.1.0            # via ipykernel, ipython
```

Comment:
I ran `pip-compile requirements.in`.

Reply:
same as if you run `make requirements` right?

Reply:
Interesting. Actually not. First of all, I had to fix `Makefile` because of `_install-piptools` that should be `install-piptools`; then `requirements.txt` file header became:
```
#
# This file is autogenerated by pip-compile
# To update, run:
#
#    make requirements.txt
#
```

I will make a commit to reflect the difference.

Reply:
I see only a few things been upgraded because of `define PIP_COMPILE`.